x/stakeibc/README.md

```diff
@@ -91,24 +91,16 @@ Governance
 
 ## Queries
 
-- `QueryInterchainAccountFromAddressRequest`
```

Comment:
why delete these queries? are they deprecated?

Reply:
The heading is queries so I thought it made more sense to just document the query type, rather than doing QueryTypeRequest and QueryTypeResponse for each